docs/contributors_guide.md

```diff
@@ -0,0 +1,141 @@
+# Contributor's guide
+
+`dbt-score` is free software, your contributions are welcome! 🚀
+
+## Reporting bugs
+
+If you encountered a bug, check the
+[issue tracker on Github](https://github.com/PicnicSupermarket/dbt-score/issues)
+to see if it is already known. If not, feel free to
+[open a new issue](https://github.com/PicnicSupermarket/dbt-score/issues/new).
+Share all relevant details, especially how to reproduce the problem.
+
+We'd love to hear from you and help make `dbt-score` as stable as we can.
+
+## Developing new features
+
+### Adding rules
+
+The linting rules bundled with `dbt-score` aim to be as generic as possible, and
+apply to a large majority of dbt projects.
+
+This is why they are not very opinionated: for example, we believe documenting
+data models is important, and hope it's not a controversial opinion ;) Of
+course, it's always possible to disable any rule.
+
+If you think a new rule should be created in `dbt-score`, feel free to
+[open an issue](https://github.com/PicnicSupermarket/dbt-score/issues/new) to
+discuss it first - this might save you some work in case maintainers don't see a
+fit.
+
+If your rule idea is not generic and applies to your own project and logic,
+`dbt-score` has been designed to fully [support custom rules](create_rules.md).
+Create as many as you need for your purposes!
+
+### Fixing bugs
+
+We love bug squashing! You can open a pull request to fix any bug you encounter
+in `dbt-score`. If the changes are large enough, refer to the next section
+first - discussing a solution in a Github issue is always a good idea to avoid
+unnecessary work and orchestrate efforts.
+
+### Adding or changing core features
+
+Before implementing or changing a new feature, we kindly ask you to
```

Comment:
I think we mention a lot of times that they should always open an issue first, becomes a little bit too much IMO